### DISCUSSION TITLE
Neck Customizations

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#199] [Feature Requests]
It would be cool to add neck customization on top of head and tail!

The specific use case that comes to mind is having the trans-right scarf be a 'neck customization' so that it can be combined with other heads!

I can imagine necklaces and stuff like that being added as other neck customizations in the future

**Comment by bvanvugt** [comment]
Yeah! Totally agree, we've talked about this internally too. 

We've got a different, equally new customization in the pipeline atm, and then this will likely follow after that.
